event_email_reminder/README.rst

```diff
@@ -45,8 +49,10 @@ Images
 Contributors
 ------------
 
-* Sergio Teruel <sergio.teruel@tecnativa.com>
-* Vicent Cubells <vicent.cubells@tecnativa.com>
+* Tecnativa <https://www.tecnativa.com>:
+  * Sergio Teruel <sergio.teruel@tecnativa.com>
+  * Vicent Cubells <vicent.cubells@tecnativa.com>
+  * Pedro M. Baeza
```

Comment:
Your email is missing.

Reply:
I don't want to put anymore my mail, hehe